warehouse/templates/manage/organization/roles.html

```diff
@@ -160,9 +160,19 @@ <h2>
         <td class="table__align-center"></td>
         <td class="table__align-right">
           {% set extra_fields %}<input name="user_id" type="hidden" value="{{ invite.user_id }}">{% endset %}
-          {% set extra_description %}{% trans user=invite.user.username %}Revoke invitation for {{ user }}{% endtrans %}{% trans %}.{% endtrans %}{% endset %}
-          {% set tooltip %}{% trans user=invite.user.username %}Revoke invitation for {{ user }}{% endtrans %}{% endset %}
-          {{ confirm_button(gettext("Revoke invite"), None, "revoke-invite-" + invite.user_id|string, None, extra_fields=extra_fields, extra_description=extra_description, action=request.route_path('manage.organization.revoke_invite', organization_name=organization.normalized_name), warning=False, modifier="--primary", tooltip=tooltip) }}
+          {% if invite.invite_status == OrganizationInvitationStatus.Pending %}
+            {% set extra_description %}{% trans user=invite.user.username %}Revoke invitation for {{ user }}{% endtrans %}{% trans %}.{% endtrans %}{% endset %}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏Looks like this is part of the original refactor, but smells odd to be to have `...{% endtrans %}{% trans %}.{% endtrans %}` - like we have to open a new tag for translating a period at the end of a sentence? This is repeated on the next if/elifs as well.

Reply:
Hm, yeah. That's wrong. Will fix while we're here. See 722f0928408f77122a468fe1c2fd453491208a8c